File: src/main/java/boj/problems/No1330.java
package boj.problems;

import java.io.*;
import java.util.StringTokenizer;

public class No1330 {
    public static void main(String[] args) throws IOException {
        BufferedReader input = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter output = new BufferedWriter(new OutputStreamWriter(System.out));

        solve(input, output);

        output.flush();
        output.close();
    }

    static void solve(BufferedReader input, BufferedWriter output) throws IOException {
        StringTokenizer st = new StringTokenizer(input.readLine());
        int a = Integer.parseInt(st.nextToken());
        int b = Integer.parseInt(st.nextToken());

        if (a < b) {
            output.write("<\n");
            return;
        }

        if (a > b) {
            output.write(">\n");
            return;
        }

        output.write("==\n");
        return;
    }
}

File: src/main/kotlin/boj/problems/No1330.kt
package boj.problems

import java.io.BufferedReader

class No1330 {
    fun solve(input: BufferedReader): String {
        val (a, b) = input.readLine().split(" ").map { it.toInt() }

        return when {
            a < b -> "<"
            a > b -> ">"
            else -> "=="
        }
    }
}
